generator/src/test/java/com/scottlogic/deg/generator/cucumber/testframework/utils/CucumberGenerationConfigSource.java

```diff
@@ -107,4 +107,9 @@ public boolean visualiseReductions() {
     public GenerationConfig.OutputFormat getOutputFormat() {
         return null;
     }
+
+    @Override
+    public GenerationConfig.OutputDestination getOutputDestination() {
+        return state.outputDestination;
```

Comment:
This method can throw, or always yield only of the known values - cucumber doesn't care about this part as it circumvents the output targets.